test/lit/validation/nn-tuples.wast

```diff
@@ -1,17 +0,0 @@
-;; Test for non-nullable types in tuples
```

Comment:
Why was this deleted?

Reply:
This tested a case that now validates fine (non-nullable entries in tuples).

I guess it could be manually rewritten to show the opposite result, if we think that's important? (the auto-updater script can't do it for us here, and I'm not sure it's worth the work)

Reply:
It would be nice to test that non-nullable tuples are ok now with the proper feature set enabled. If we only have one RUN line, this could be handled by the auto update script.

Reply:
Ok, I restored the test and made it just check validation.